test/functionalTest/cases/0000_ngsild/ngsild_registration_get_error_handling.test

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Handle errors of Context Source Registrations GET
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. GET  /ngsi-ld/v1/csourceRegistrations GET limit = 0
+# 02. GET  /ngsi-ld/v1/csourceRegistrations GET limit = 1001
+# 03. GET  /ngsi-ld/v1/csourceRegistrations GET limit = abc 
+#
+
+echo "01. GET  /ngsi-ld/v1/csourceRegistrations GET limit = 0"
+echo "======================================================="
+orionCurl --url /ngsi-ld/v1/csourceRegistrations?limit=0
+echo
+echo
+
+
+echo "02. GET  /ngsi-ld/v1/csourceRegistrations GET limit = 1001"
+echo "=========================================================="
+orionCurl --url /ngsi-ld/v1/csourceRegistrations?limit=1001
+echo
+echo
+
+
+echo "03. GET  /ngsi-ld/v1/csourceRegistrations GET limit = abc"
+echo "========================================================="
+orionCurl --url /ngsi-ld/v1/csourceRegistrations?limit=abc
+echo
+echo
+
+
+--REGEXPECT--
+01. GET  /ngsi-ld/v1/csourceRegistrations GET limit = 0
+=======================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 132
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "details": "The reason for this error is unknown",
+    "title": "Unknown Error",
+    "type": "http://uri.etsi.org/ngsi-ld/errors/InternalError"
+}
+
+
+02. GET  /ngsi-ld/v1/csourceRegistrations GET limit = 1001
+==========================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 132
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "details": "The reason for this error is unknown",
+    "title": "Unknown Error",
+    "type": "http://uri.etsi.org/ngsi-ld/errors/InternalError"
+}
+
+
+03. GET  /ngsi-ld/v1/csourceRegistrations GET limit = abc
+=========================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 132
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "details": "The reason for this error is unknown",
+    "title": "Unknown Error",
```

Comment:
And here, same same.
One single error message is enough - "Invalid limit for pagination"
And, the error type should be Bad Request, not Internal Error

Reply:
Error message changed in 147df5b - implementation pending though ...

Reply:
Implementation done in 70b1772